graph.py

```diff
@@ -64,7 +68,7 @@ def update_nodes(self, nodes):
     def update_links(self, links):
         """Update all links inside the graph."""
         for link in links.values():
-            if link.is_active():
+            if link.status == EntityStatus.UP:
```

Comment:
Link should not be `active` if the `status` is not `UP`. If that is happening, we should investigate the cause.

Reply:
@ajoaoff this is a same as the change for interface.status and node.status, stronger guarantees that a disabled link won't end up being added to the graph, it's possible to have a `enabled: false` link with `active: true` state.

Reply:
I understand it, just pointing that if a link can be active while disabled than something is not working properly and we might investigate why it is happening.

Reply:
Got it @ajoaoff. [Looking into EP013 again (several years since we participated in that spec though)](https://github.com/kytos-ng/kytos/blob/master/docs/blueprints/EP013.rst) and [how it was implemented accordingly](https://github.com/kytos-ng/kytos/blob/master/kytos/core/common.py#L45-L51), it seems that the intention is indeed that `status` is supposed to be the way to check if both the `_active` and `_enable` state should be taken into account 

Your point makes sense, Antonio, like, why is something admin down but still active? Based on the current implementation, I think it was to simplify how the `_active` state currently maps one to one to every port status (at least for `Interface` and `Link`), and if admin down were also to affect `_active`, then we'd have to have another attribute or persistent attribute to try to restore from it (which also would have to taken into account if `_active` hasn't changed again for new OpenFlow events), right? For example, a case where a link a up, and then an operator shuts it down, but how we'd restore the `_active` state again, we'd need this extra attr to consider in the logic. 

1 - Maybe this is something to document better? 

2 - Or if operators feel like it's unacceptable to have `enabled: false` and `active: true` in some cases, assuming they are aware of the semantics, then we'd have to implement this support, what do you think @ajoaoff? Although as it is, I think it's simpler to maintain.

Reply:
@ajoaoff I'll go ahead and merge this PR since it's been approved, thanks for reviewing, let's continue on this thread here to make sure if any other next step is covered.